templates/ocs-deployment/private.yaml

```diff
@@ -0,0 +1,9 @@
+###################################################################################################################
+# Create a private namespace 
+###################################################################################################################
+---
+# Namespace used to test individual scripts. 
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ cluster_namespace }}
```

Comment:
If this file is not used as a part of the actual OCS deployment it doesn't belong in `templates/ocs-deployment`. We should create another directory under `templates/` to house this new file.

Reply:
I added a new directory (test-deployment) and changed the create_oc_resource call.